### ISSUE TITLE
Producer usage

### ISSUE THREAD

**Issue by tphummel** [datascienceinc/accord#9]
This project looks perfect for our needs. I understand the usage as a test double on the consumer side. I'm less clear on the usage for running contract tests on the producer side. The readme shows creating one or more `accord` blocks that can reference git, s3, http sources for contract hcl files. What might a convention look like in a producer project for asserting that consumer contracts are being upheld? And what would the corresponding command to run the consumer tests be?

**Comment by ChrisMcKenzie** [comment]
Yeah I think I kind of slacked off on the docs for the producer test side of this. I am going to make this a TODO for docs!
